src/Protocol/Diaspora.php

```diff
@@ -2148,9 +2149,9 @@ private static function receiveParticipation(array $importer, $data)
 				continue;
 			}
 			if ($comment['verb'] == ACTIVITY_POST) {
-				$cmd = $comment['self'] ? 'comment-new' : 'comment-import';
+				$cmd = $comment['self'] ? Delivery::COMMENT : 'comment-import';
 			} else {
-				$cmd = $comment['self'] ? 'like' : 'comment-import';
```

Comment:
just to be sure .. so `like` is now intenationally replaced with `activtiy-new` .. I don't understand the content well enough, but this is something I noticed

Reply:
Yeah. I just unified this. But in fact the content here doesn't matter much. It is important for some special stuff. But except from this we could put everything there.